store/src/main/java/com/nytimes/android/external/store3/pipeline/Async.kt

```diff
@@ -0,0 +1,88 @@
+package com.nytimes.android.external.store3.pipeline
+
+import java.util.Arrays
+
+/**
+ * The T generic is unused for some classes but since it is sealed and useful for Success and Fail,
+ * it should be on all of them.
+ *
+ * Complete: Success, Fail
+ * ShouldLoad: Uninitialized, Fail
+ */
+sealed class Async<out T>(val complete: Boolean, val shouldLoad: Boolean) {
```

Comment:
looks very similar to StoreResponse, is it intended to replace it?

Reply:
ignore for now